src/metaopt/tests/mongodb_test.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+
+from metaopt.io.database.mongodb import MongoDB
+
+class MongoDB_test():
+    def __init__(self):
+        self.db = MongoDB()
+        self.experiment = 'genetic'
+        self.trial = 'trial1'
+        self.query_test = {
+            "name": "decoding_layer",
+            "type": "int",
+            "value": "300",
+        }
+        self.field_filters = {
+            "name": "decoding_layer"
+        }
+
+        self.update_query_test = {
+            "$set": {
+                "value": "400"
+            }
+        }
+
+    def mongodb_connection_test(self):
+        conn = self.db.check_connection()
+        return conn
+
+    def mongodb_insert_test(self):
+        result = self.db.insert(self.experiment, self.trial, self.query_test, field_filters=None)
```

Comment:
ok I see here what trial_field was intended to be. I'd prefer to combine them inside a single dict inside experiments otherwise they kind of *pollute* the experiment row.

Reply:
I am not sure that I understand what you mean. Could you give 2 cases?

Reply:
```
{ name: dummy,
  # some other properties,
  trial_1: {
      # properties of trial_1
   },
  # ...
  trial_n: {
      # properties of trial_n
   }
  # some other properties,
}
```

vs

```
{ name: dummy,
  # some other properties,
  trials: [
      {
          id: 1, 
          # properties of trial #1 
      },
      # ...
      trial_n: {
          # properties of trial_n
      }
   },
  # some other properties,
}
```

Trials could be either a list or a dictionary